x-pack/test/accessibility/apps/ml.ts

```diff
@@ -10,6 +10,7 @@ import { FtrProviderContext } from '../ftr_provider_context';
 export default function ({ getService }: FtrProviderContext) {
   const a11y = getService('a11y');
   const ml = getService('ml');
+  const kibanaServer = getService('kibanaServer');
```

Comment:
need this?

Reply:
Nopes. Didn't notice and then type error. My env needs fixing. Thanks!